coderd/database/migrations/000048_workspace_resource_icon.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE workspace_resources
```

Comment:
This migration will fail because it conflicts with the `userdelete` one on `main`. You'll have to up the numbers manually.